### PR TITLE
hypershift input changes

### PR DESCRIPTION
**Pull request by mukrishn**
### Description
1. Changes to Hypershift MC and HC install size, 
2. Operator configuration to enable metrics-set
3. Updated workload to use new P75 and P90
4. Updated release names per playground, added branch in the cluster name

### CODE REVIEW

**Review by dry923 — APPROVED** [review]
removed LGTM given our slack convo w/ you splitting the dags

**Review by venkataanil — APPROVED** [review]
LGTM

File: dags/openshift_nightlies/config/install/hypershift/none-type.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
     "cluster_install_method": "osd",
-    "ocm_cli_version": "master",
```
Comment (morenod) [inline dags/openshift_nightlies/config/install/hypershift/none-type.json:3]: I use to point to "container" on rosa when I dont want to use master, so we dont have on the default behavior. 

https://github.com/cloud-bulldozer/airflow-kubernetes/blob/master/dags/openshift_nightlies/scripts/install/rosa.sh#L201

It is better than comparing to null, which could be confuse

**Review by morenod — CHANGES_REQUESTED** [review]

File: dags/openshift_nightlies/config/install/hypershift/none-type.json (hunk 1 above)
Comment (mukrishn) [reply dags/openshift_nightlies/config/install/hypershift/none-type.json:3]: Sure I will set a default value instead of `null`.

**Review by morenod — APPROVED** [review]
LGTM

**Pull request merged**